services/sourceforge/sourceforge-commit-count.service.js

```diff
@@ -0,0 +1,53 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  commit_count: Joi.number().required(),
+})
+
+export default class SourceforgeCommitCount extends BaseJsonService {
+  static category = 'activity'
+
+  static route = {
+    base: 'sourceforge/commit-count',
+    pattern: ':project',
+  }
+
+  static examples = [
+    {
+      title: 'SourceForge commit count',
+      namedParams: {
+        project: 'guitarix',
+      },
+      staticPreview: this.render({
+        commitCount: 1365,
+      }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'commit count' }
+
+  static render({ commitCount }) {
+    return {
+      message: commitCount,
+      color: 'blue',
+    }
+  }
+
+  async fetch({ project }) {
+    return this._requestJson({
+      url: `https://sourceforge.net/rest/p/${project}/code`,
+      schema,
+      errorMessages: {
+        404: 'project not found',
+      },
+    })
+  }
```

Comment:
We're repeating this same fetch method in every class.

I think we could usefully split our a base class

```js
class BaseSourceForgeService extends BaseJsonService {
  async fetch({ project, schema }) {
    return this._requestJson({
      url: `https://sourceforge.net/rest/p/${project}/code`,
      schema,
      errorMessages: {
        404: 'project not found',
      },
    })
  }
}
```

and have all of our sourceforce services extend `BaseSourceForgeService` to DRY this up a bit.